flake.nix

```diff
@@ -70,6 +70,12 @@
               cargo fmt --check && echo "\`cargo fmt\` is OK"
               cargo clippy -- --deny warnings && echo "\`cargo clippy\` is OK"
             '';
+          }
+          //
+          {
+            generate-config = shell: final.runCommand "nix-your-shell-config" {} ''
```

Comment:
We might want this to have a more unique name in case downstream users apply this overlay on their entire nixpkgs. Maybe we can put it in `nix-your-shell.passthru.generate-config`?

https://nixos.org/manual/nixpkgs/stable/#special-variables

Reply:
I've put `generate-config` to `passthru` property. Also tested that it works as before